content/javascript/concepts/bitwise-operators/bitwise-operators.md

```diff
@@ -37,6 +37,19 @@ console.log(19 & 7); // Output: 3
 // 00011 =  3
 ```
 
+## Codebyte Example
+
+The following example illustrates the use of Bitwise `&` operator:
+
+```codebyte/javascript
+console.log(-5 & 12); // Output: 8
+
+// 11111111111111111111111111111011 = -5
+// 00000000000000000000000000001100 = 12
+// 00000000000000000000000000001000 = 8
```

Comment:
```suggestion
console.log(-5 & 12); 
```
No need to add comments in the codebyte as the purpose of a codebyte is to let the reader try the code himself/herself. Remove comments from all the codebytes.

Reply:
Thanks for your review. Changes have been effected! 😊